dwave/system/utilities.py

```diff
@@ -89,3 +116,114 @@ def get(name, default=False):
     @classproperty
     def hss_solver_config_override(cls):
         return cls.get('hss_solver_config_override')
+
+
+def anneal_schedule_with_offset(
+        anneal_offset: float = 0.0,
+        anneal_schedule: Union[np.typing.ArrayLike, List, None] = None,
```

Comment:
See `list` over `List` above. But `Sequence` is usually recommended over list. If you prefer list, you might want to add item type: `list[float]` - or `list[list[float]]` for the complete schedule.